tests/risks/test_low_birth_weight_and_short_gestation.py

```diff
@@ -32,3 +39,102 @@ def test_parsing_lbwsg_descriptions(description, expected_weight_values, expecte
     assert weight_interval.right == expected_weight_values[1]
     assert age_interval.left == expected_age_values[0]
     assert age_interval.right == expected_age_values[1]
+
+
+def test_lbwsg_risk_effect_rr_pipeline(
+    base_config, base_plugins, mocker, mock_rr_interpolators
+):
+
+    risk = LBWSGRisk()
+    lbwsg_effect = LBWSGRiskEffect("cause.test_cause.cause_specific_mortality_rate")
+
+    # Add mock data to artifact
+    categories = {
+        "cat81": "Neonatal preterm and LBWSG (estimation years) - [28, 30) wks, [2500, 3000) g",
+        "cat82": "Neonatal preterm and LBWSG (estimation years) - [28, 30) wks, [3000, 3500) g",
+    }
+    # Create exposure with matching demograph index as age_bins
+    age_bins = make_age_bins()
+    agees = age_bins.drop(columns="age_group_name")
+    exposure_data = make_categorical_exposure_data(agees)
+
+    # Add data dict to add to artifact
+    data = {
+        f"{risk.name}.exposure": exposure_data,
+        f"{risk.name}.population_attributable_fraction": 0,
+        f"{risk.name}.categories": categories,
+        f"{risk.name}.relative_risk_interpolator": mock_rr_interpolators,
+    }
+
+    # Only have neontal age groups
+    age_start = 0.0
+    age_end = 28 / 365.0
```

Comment:
Should include some simulants older than 28 days and show that their RRs are 1.0